cwod_site/bioguide/management/commands/import_roles.py

```diff
@@ -17,23 +16,28 @@ def handle(self, *args, **options):
         except IndexError:
             congress = None
         bioguide_ids = Legislator.objects.order_by('bioguide_id').values_list('bioguide_id', flat=True).distinct()
+
+        s = requests.Session()
+        s.headers.update({'X-API-Key': settings.PROPUBLICA_API_KEY})
         for bioguide_id in bioguide_ids:
             # if LegislatorRole.objects.filter(bioguide_id=bioguide_id).count() > 0:
             #     continue
 
-            url = 'http://api.nytimes.com/svc/politics/v3/us/legislative/congress/members/%s.json?api-key=%s' % (bioguide_id, settings.NYT_API_KEY)
+            url = 'https://api.propublica.org/congress/v1/members/{}.json'.format(bioguide_id)
```

Comment:
Eventually we should try to consolidate all API endpoint templates in one place.